pootle/checks.py

```diff
@@ -547,3 +547,26 @@ def check_mysql_timezones(app_configs=None, **kwargs):
             if converted_now is None:
                 errors.append(missing_mysql_timezone_tables)
     return errors
+
+
+@checks.register()
+def check_mysql_collation(app_configs=None, **kwargs):
+    from django.db import connection
+
+    non_utf8_collation_and_character_set = checks.Critical(
```

Comment:
Break out collation and charset, the error is cheap and this doesn't help anyone actually understand which one is wrong.